traffic_ops/app/lib/API/Configs/ApacheTrafficServer.pm

```diff
@@ -2574,11 +2574,14 @@ sub build_remap_line {
 	if ( defined( $remap->{edge_header_rewrite} ) ) {
 		$text .= " \@plugin=header_rewrite.so \@pparam=" . $remap->{hdr_rw_file};
 	}
-	if ( $remap->{signing_algorithm} eq "url_sig" ) {
-		$text .= " \@plugin=url_sig.so \@pparam=url_sig_" . $remap->{ds_xml_id} . ".config";
-	}
-	elsif ( $remap->{signing_algorithm} eq "uri_signing" ) {
-		$text .= " \@plugin=uri_signing.so \@pparam=uri_signing_" . $remap->{ds_xml_id} . ".config";
+	if ( defined( $remap->{signing_algorithm}))
+	{
```

Comment:
You are breaking the formatting of the surrounding code by putting your curly on a new line.